www/front_src/src/Header/userMenu/index.tsx

```diff
@@ -135,10 +135,19 @@ const useStyles = makeStyles((theme) => ({
   },
   icons: {
     alignItems: 'center',
+    borderLeft: '1px solid white',
```

Comment:
`white` it's not possible tu replace with `theme.palette.common.white` ?
to define border and another prop with borderColor ?